packages/@react-spectrum/provider/docs/Provider.mdx

```diff
@@ -76,6 +77,26 @@ Another important setting for your application is the locale. By default, React
 
 To access the current locale anywhere in your application, see the [useLocale](https://react-spectrum.adobe.com/react-aria/useLocale.html) hook.
 
+### Breakpoints
+
+The Provider component can also be used to set the responsive breakpoints for your application, which are used by [layout components](layout.html#responsive-layout) and [style props](styling.html#responsive-styles) of all components within the Provider. By default, React Spectrum provides 5 breakpoints, which correspond to common device resolutions:
+
+* S – 640px
+* M - 768px
+* L – 1024px
+* XL - 1280px
+* XXL - 1536px
+
+These breakpoints can be overridden by passing the `breakpoints` prop to the `Provider` component. Any components within the Provider will consume these breakpoints and apply their style props accordingly. By default, React Spectrum uses t-shirt style names for breakpoints, but you can name custom breakpoints however you like.
+
+This example specifies two breakpoints: `tablet` and `desktop`. It displays a [View](View.html) that changes color depending on which breakpoint is matched: green by default, blue at the `tablet` breakpoint, and magenta at the `desktop` breakpoint. See the docs for [style props](styling.html#responsive-styles) for more details.
+
+```tsx example
+<Provider theme={theme} breakpoints={{tablet: 640, desktop: 1024}}>
+  <View height="size-1000" backgroundColor={{base: 'celery-600', tablet: 'blue-600', desktop: 'magenta-600'}} />
+</Provider>
+```
```

Comment:
I think this example is supposed to render something right? Looks like the examples on the Provider page are broken on the docs site too, this older docs build shows stuff: https://reactspectrum.blob.core.windows.net/reactspectrum/2061244489fe142e2b26da3158fdafb43fac1bba/docs/react-spectrum/Provider.html
I'll dig tmrw when I get a change

Reply:
sent you a slack message, we're hoping it's not broken in the prod build atm before you get sucked into it